lib/geolocation/simulator.js

```diff
@@ -14,27 +14,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+'use strict';
+
 var console = require('console')
 
 function Simulator () {
-   throw("TODO: unimplemented");
+  this.value = {
+    latitude: 0,
+    longitude: 0
+  };
 }
 
 Simulator.prototype.read = function (callback) {
-  throw("TODO: unimplemented");
-  if (callback) return callback(null, this.value)
+  this.value.latitude = Math.random() * 180 - 90;
+  this.value.longitude = Math.random() * 360 - 180;
+  if (callback) return callback(null, this.value);
 }
 
-module.exports = Simulator
+module.exports = Simulator;
```

Comment:
this change should be in lint patches no ?

Reply:
Ah, you're right - I added that on my own accord, not in response to ESLint warnings; that's why I missed it in the rebase. Fixed now (along with a few other semi-colons.)